coast/CLIMATOLOGY.py

```diff
@@ -76,10 +84,122 @@ def make_climatology(
                 ds = ds.drop_vars("clim_mean_ones_tmp")
```

Comment:
It looks like ds isn't used at all after this point. Is this necessary?

Reply:
Removed as part of the revamp of the method to use xarray's built in mean() functionality.